spec/jobs/evss/update_claim_from_remote_job_spec.rb

```diff
@@ -48,5 +48,19 @@
         expect(tracker.get_single_status).to eq('FAILED')
       end
     end
+
+    context 'when an active record error occurs' do
+      it 'should set the status to FAILED' do
```

Comment:
Perhaps add an `:aggregate_failures` to this guy, for the multiple expectations?  i.e. 

```ruby
it 'should set the status to FAILED', :aggregate_failures do
```

Reply:
Looked up `:aggregate_failures`, thats very convenient!